--- mesonbuild/modules/gnome.py.glib	2017-02-21 20:45:17.187843787 -0500
+++ mesonbuild/modules/gnome.py	2017-02-21 20:46:18.062188395 -0500
@@ -37,7 +37,7 @@
 #
 # https://github.com/ninja-build/ninja/issues/1184
 # https://bugzilla.gnome.org/show_bug.cgi?id=774368
-gresource_dep_needed_version = '>= 2.52.0'
+gresource_dep_needed_version = '>= 2.50.0'
 
 native_glib_version = None
 girwarning_printed = False
